lib/util.js

```diff
@@ -134,14 +134,16 @@ const uploadExtension = (id, version, crxFile) => {
     }
   })
   const componentFilename = `extension_${version.replace(/\./g, '_')}.crx`
+  const tag = `tag=${name.replace(/\s/g, '-')}/${version.replace(/\s/g, '-')}`
```

Comment:
Can we guarantee name and version combination is unique?

Ex, NTP SI uses hundreds of components because it uses unique component per region.
and all components use same name (`Brave NTP sponsored images`).
If you want, I can update their name like `Brave NTP sponsored images (US)`.

and not sure all other components set unique name in manifest.

Reply:
I think it is okay for `Brave NTP sponsored images` - We would like the total number of downloads for the component irrespective of the locale. cc: @hspencer77 @bbondy

Reply:
We can't differentiate between GB's SI component (1.0.61) and CA's 1.0.61.
Both are different components but will be calculated as same component w/o locale.
Is it ok?

Reply:
I believe its okay. @bbondy @hspencer77 - Let me know if you feel otherwise.